cylc/flow/cfgspec/suite.py

```diff
@@ -327,7 +327,7 @@
             for this, it will be assumed to be local time, or in UTC if
             :cylc:conf:`flow.cylc[scheduler]UTC mode` is set, or in the time
             zone determined by
-            :cylc:conf`flow.cylc[scheduler][cycle point time zone]`.
+            :cylc:conf:`flow.cylc[scheduler]cycle point time zone`.
```

Comment:
Setting referenced as a section.

Didn't fail the docs build because the reference was corrupted (missing the trailing `:`).